english/plugh/maze1/maze1.md

```diff
@@ -0,0 +1,14 @@
+You are in a maze of twisty little passages, all different.
+
+Which way should we go?
+
+[north](maze1.md)
+[south](maze1.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.